components/bl-credit-card-component/src/card-preview.js

```diff
@@ -0,0 +1,110 @@
+import { useMemo } from 'react';
+
+const InitialCardProps = {
+  expirationLocaleHeader: 'valid thru',
+  namePlaceholder       : 'YOUR NAME HERE',
+  issuer                : 'unknown',
+  numberLength          : 16,
+  numberMaxLength       : 19,
+};
+
+const NumberSpaces = { shortFormat: [4, 11], defaultFormat: [4, 9, 14] };
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export function CardPreview(props) {
+  const { cardNumber, expiry, cvc, focused, name, card, cvcVisibility } = props;
+
+  const { creditCardNumber, expirationDate, issuer } = useCardPreview(cardNumber, expiry, card);
+
+  const currentCVC = useMemo(() => cvcVisibility ? cvc : cvc.replace(/\d/g, '*'), [cvc, cvcVisibility]);
+
+  return (
+    <div className="card-container">
+      <div className={ cn('card', `card-${ issuer }`, { 'card-flipped': focused === 'cvc' && issuer !== 'amex' }) }>
+        <div className="card-front">
+          <div className="card-background"/>
+          <div className="issuer"/>
+          <div className={ cn('cvc-front', { 'focused': focused === 'cvc' }) }>{ currentCVC }</div>
```

Comment:
focused - is boolean variable name
I assume there should be focusedFieldName / focusedField